commons/src/lib.rs

```diff
@@ -104,6 +116,10 @@ pub fn validate_content_type(
     let full_type = header::HeaderValue::from_static(content_type);
     let wildcard = header::HeaderValue::from_static("*");
     let double_wildcard = header::HeaderValue::from_static("*/*");
+    let mut cinci_versions: Vec<actix_web::http::HeaderValue> = CINCI_VERSION
```

Comment:
nit: this local variable definitely doesn't need `cinci_`, because any readers will know they're looking at Cincinnati code.